modules/core/src/v2/environments.ts

```diff
@@ -57,6 +57,7 @@ export type EnvironmentName =
   | 'staging'
   | 'test'
   | 'dev'
+  | 'testnet'
```

Comment:
maybe something a bit more verbose, devtestnet? testnet feels like its referring to test